### PR TITLE
Added UseScroller property to NavigationView to control IsDynamicScrollViewerEnabledProperty

### PR DESCRIPTION
**Pull request by Akizon77**
- [x] Feature

Added UseScroller property to NavigationView to control IsDynamicScrollViewerEnabledProperty of NavigationViewContentPresenter(NVCP).(I conflicted with NVCP DynamicScrollViewer when designing ItemsControls. I tried many methods but failed to solve it (including adding UseScroller attribute, the state was not passed to NVCP). Finally, I found IsDynamicScrollViewerEnabled = ScrollViewer.GetCanContentScroll() by downloading the source code and debugging continuously;(NavigationViewContentPresenter.cs,Line 177) only to find out that you have considered this situation. Navigation documentation only describes how to navigate, no more detailed information, plus I have not considered the need to add AP to the Page, so I spent the whole day debugging and thinking.

Although I am a beginner of WPF, I think Frame should default without ScrollViewer. Previously, when doing some Win UI3 projects, Frame is without ScrollViewer, just a Frame.

The WPF UI project is one of my favorite open source projects, and I learned a lot from its source code (there is almost no systematic C#teaching in our country, let alone WPF). I hope this project gets better and better, and I hope to contribute to the community with my limited knowledge.

### CODE REVIEW

File: src/Wpf.Ui/Controls/NavigationView/NavigationViewContentPresenter.cs [hunk 1]
```diff
@@ -167,7 +167,7 @@ protected virtual void OnNavigated(NavigationEventArgs eventArgs)
             return;
         }
 
-        IsDynamicScrollViewerEnabled = ScrollViewer.GetCanContentScroll(dependencyObject);
+        //IsDynamicScrollViewerEnabled = ScrollViewer.GetCanContentScroll(dependencyObject);
```
Comment (pomianowski) [inline src/Wpf.Ui/Controls/NavigationView/NavigationViewContentPresenter.cs:170]:
```suggestion
```

File: src/Wpf.Ui/Controls/NavigationView/NavigationView.xaml [hunk 2]
```diff
@@ -24,10 +24,10 @@
         <Setter Property="Padding" Value="0" />
         <Setter Property="OpenPaneLength" Value="320" />
         <Setter Property="CompactPaneLength" Value="40" />
-        <Setter Property="ScrollViewer.CanContentScroll" Value="False" />
+        <!--<Setter Property="ScrollViewer.CanContentScroll" Value="False" />
```
Comment (pomianowski) [inline src/Wpf.Ui/Controls/NavigationView/NavigationView.xaml:27]: If unnecessary, it is better to remove it.

File: src/Wpf.Ui/Controls/NavigationView/NavigationView.Properties.cs [hunk 3]
```diff
@@ -15,6 +15,22 @@ namespace Wpf.Ui.Controls;
 
 public partial class NavigationView
 {
+    /// <inheritdoc/>
```
Comment (pomianowski) [inline src/Wpf.Ui/Controls/NavigationView/NavigationView.Properties.cs:18]: This `inheritdoc` does not lead to anything, it has no documentation in `INavigationView`. Additionally, it has no equivalent in WPF or Win UI. Which makes the name quite unintuitive.

**Review by pomianowski — CHANGES_REQUESTED** [review]

**Comment by IvanDmitriev1** [comment]
This functionality already exist and used in `IconsPage` https://github.com/lepoco/wpfui/blob/bdff9f93d108ab3ec27c89096d0ccfb833a75e8b/src/Wpf.Ui.Gallery/Views/Pages/DesignGuidance/IconsPage.xaml#L20C42-L20C42

**Pull request closed without merging**